nexus/src/app/organization.rs

```diff
@@ -30,30 +53,6 @@ impl super::Nexus {
         self.db_datastore.organization_create(opctx, new_organization).await
     }
 
-    pub async fn organization_fetch(
-        &self,
-        opctx: &OpContext,
-        organization_name: &Name,
-    ) -> LookupResult<db::model::Organization> {
-        let (.., db_organization) = LookupPath::new(opctx, &self.db_datastore)
-            .organization_name(organization_name)
-            .fetch()
-            .await?;
-        Ok(db_organization)
-    }
-
-    pub async fn organization_fetch_by_id(
-        &self,
-        opctx: &OpContext,
-        organization_id: &Uuid,
-    ) -> LookupResult<db::model::Organization> {
-        let (.., db_organization) = LookupPath::new(opctx, &self.db_datastore)
-            .organization_id(*organization_id)
-            .fetch()
-            .await?;
-        Ok(db_organization)
-    }
```

Comment:
@davepacheco you mentioned in #1957 that we could probably just ditch the fetch endpoints now that we're passing around lookups. I've been doing that.